### DISCUSSION TITLE
Errors and suspicious code fragments in .NET 6 sources

### DISCUSSION THREAD

**Discussion by sirinath** [dotnet/runtime#63355] [General] [answered]
Are the issues highlighted here valid requiring a fix: https://pvs-studio.com/en/blog/posts/csharp/0903/

**Comment by stephentoub** [comment, marked as answer]
Dup of https://github.com/dotnet/runtime/issues/63149, which has been closed.

**Reply by danmoseley** [reply]
@sirinath if you find other bugs in the latest sources with this or another tool, we would welcome an issue (one issue per defect).
